vector/src/main/java/im/vector/app/features/home/room/detail/RoomDetailFragment.kt

```diff
@@ -1707,14 +1751,18 @@ class RoomDetailFragment @Inject constructor(
         onUrlClicked(url, url)
     }
 
-    override fun onPreviewUrlCloseClicked(eventId: String, url: String) {
-        roomDetailViewModel.handle(RoomDetailAction.DoNotShowPreviewUrlFor(eventId, url))
+    override fun onPreviewUrlCloseClicked(eventId: String, fileUrl: String) {
+        roomDetailViewModel.handle(RoomDetailAction.DoNotShowPreviewUrlFor(eventId, fileUrl))
```

Comment:
Why this change?

Reply:
Done. (I thought I already discarded this chunk)